sdk/event_grid/src/rfc3339_utc_serializer.rs

```diff
@@ -0,0 +1,58 @@
+use chrono::{DateTime, SecondsFormat, Utc};
+use serde::{self, Deserialize, Deserializer, Serializer};
+
+pub fn serialize<S>(date: &DateTime<Utc>, serializer: S) -> Result<S::Ok, S::Error>
```

Comment:
I think RFC 3339 serialization is actually the default format for chrono

Reply:
Oooh, cool, thanks I'll look at that!

Reply:
Thanks for the feedback! I've removed the serializer and used that feature. I also added a test just to check that the date-time was being serialized correctly so that if this feature changes in the future in a breaking way, the tests will fail.